namer/namer.cc

```diff
@@ -1791,7 +1787,7 @@ ast::ParsedFilesOrCancelled defineSymbols(core::GlobalState &gs, vector<SymbolFi
     const auto &epochManager = *gs.epochManager;
     uint32_t count = 0;
     for (auto &fileFoundDefinitions : allFoundDefinitions) {
-        prodCounterAdd("types.input.founddefs.total", fileFoundDefinitions.names->definitions().size());
+        prodCounterAdd("types.input.foundmethods.total", fileFoundDefinitions.names->methods().size());
```

Comment:
I wasn't sure whether it was faster. We already have some batching logic so that we don't send all counters immediately. The loop over all found definitions might be so slow that it becomes pinned on memory latency because the inner loop is only doing some addition.

Since I'm lazy and I think that both ways are likely equally fast, I'm inclined to keep it as is, but if you want to test and rewrite it that's fine by me.

Reply:
Even with batching logic, you still have to look up "where am I keeping this counter?", which is a hashtable lookup...every iteration.  Wouldn't it be faster to only do that lookup once?

I don't understand the argument from memory latency...the proposed increment of `total` is happening in the same loop, not a separate one.  Where does memory latency matter?